GetFed/GetFed/Controllers/FoodEntryViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  FoodEntryViewController.swift
+//  GetFed
+//
+//  Created by Britney Smith on 12/4/18.
+//  Copyright © 2018 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+class FoodEntryViewController: UIViewController {
+    
+    // MARK - Outlets
+    @IBOutlet var foodTextField: UITextField!
+    @IBOutlet var brandTextField: UITextField!
+    @IBOutlet var proteinTextField: UITextField!
+    @IBOutlet var carbsTextField: UITextField!
+    @IBOutlet var fatTextField: UITextField!
+    @IBOutlet var foodLabel: UILabel!
+    @IBOutlet var brandLabel: UILabel!
+    @IBOutlet var proteinLabel: UILabel!
+    @IBOutlet var carbsLabel: UILabel!
+    @IBOutlet var fatLabel: UILabel!
+    @IBOutlet var scrollView: UIScrollView!
+    
+    // MARK - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        scrollView.contentSize = CGSize(width: self.view.frame.width, height: self.view.frame.height + 100)
```

Comment:
I know why you did this, but it really isn't necessary. If you encapsulate all of your views inside the scroll view in a view, and then set that view's width to be constrained to the scrollview's superview's width, it should probably determine the scrollable area.

Reply:
Change applied.